<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wifiListWidget</class>
 <widget class="QWidget" name="wifiListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#wifiListWidget
{
background-color: rgb(73, 92, 105);
}</string>
  </property>
  <widget class="QListWidget" name="listWidgetWIFI">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>301</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWifi">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetButton" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widgetKeyboard" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>281</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
